ARTIFACT_INVENTORY.md

```diff
@@ -25,15 +25,15 @@ registry/repository/image_name:version
 | Metrics | Alertmanager | quay.io/prometheus/alertmanager:v0.25.0|
 | Metrics | BusyBox | docker.io/library/busybox:1.31.1|
 | Metrics | Ghostunnel | docker.io/ghostunnel/ghostunnel:v1.7.1|
-| Metrics | Grafana | docker.io/grafana/grafana:9.5.5|
+| Metrics | Grafana | docker.io/grafana/grafana:10.0.3|
 | Metrics | Ingress-NGINX Controller | registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20221220-controller-v1.5.1-58-g787ea74b6|
-| Metrics | Kube State Exporter | k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.8.2|
-| Metrics | Node Exporter | quay.io/prometheus/node-exporter:v1.5.0|
-| Metrics | Prometheus | quay.io/prometheus/prometheus:v2.44.0|
-| Metrics | Prometheus Operator | quay.io/prometheus-operator/prometheus-operator:v0.65.1|
-| Metrics | Prometheus Operator (config reloader) | quay.io/prometheus-operator/prometheus-config-reloader:v0.65.1|
-| Metrics | Prometheus Pushgateway | docker.io/prom/pushgateway:v1.5.1|
-| Metrics | Sidecar | quay.io/kiwigrid/k8s-sidecar:1.24.0|
+| Metrics | Kube State Exporter | k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.9.2|
+| Metrics | Node Exporter | quay.io/prometheus/node-exporter:v1.6.0|
+| Metrics | Prometheus | quay.io/prometheus/prometheus:v2.45.0|
+| Metrics | Prometheus Operator | quay.io/prometheus-operator/prometheus-operator:v0.66.0|
+| Metrics | Prometheus Operator (config reloader) | quay.io/prometheus-operator/prometheus-config-reloader:v0.66.0|
+| Metrics | Prometheus Pushgateway | docker.io/prom/pushgateway:v1.6.0|
+| Metrics | Sidecar | quay.io/kiwigrid/k8s-sidecar:1.24.6|
```

Comment:
- Incorrect version.
- Please change "Sidecar" to "Auto-load Sidecars (Grafana)"